spec/tests/techniques/technique_sharedFiles.rb

```diff
@@ -0,0 +1,11 @@
+require 'spec_helper'
+basename = $params['BASENAME']
```

Comment:
You could guess basename in ruby, example https://stackoverflow.com/questions/374326/how-to-get-filename-without-extension-from-file-path-in-ruby